packages/intl-enumerator/data/units.ts

```diff
@@ -0,0 +1,190 @@
+export type Unit = typeof units[number]
+
+export const units = [
+  '100-kilometer',
+  'acre',
+  'acre-foot',
+  'ampere',
+  'arc-minute',
+  'arc-second',
+  'astronomical-unit',
+  'atmosphere',
+  'bar',
+  'barrel',
+  'bit',
+  'british-thermal-unit',
+  'bushel',
+  'byte',
+  'calorie',
+  'candela',
```

Comment:
this list seems odd because a bunch of units here are not sanctioned https://tc39.es/ecma402/#table-sanctioned-simple-unit-identifiers